### PR TITLE
Instructor: view results: add visibility tooltip to feedback response comments #5597

### LINKED ISSUE #5597: Instructor: view results: add visibility tooltip to feedback response comments
Currently, the visibility of student comments can be easily seen via the use of an eye icon:

commentsvisibility

However, this icon is not present on feedback response comments, making it very difficult to figure out who is able to see the comment.

Edit: this only applies to instructorFeedbackResultsPage and instructorStudentRecordsPage, as instructorCommentsPage correctly displays the visibility settings

### PR DESCRIPTION
**Pull request by Mynk96**
Fixes #5597

### CODE REVIEW

**Comment by Mynk96** [comment]
UI changes:-

`InstructorStudentRecordsPage`

studentvis


`InstructorFeedbackResultsPage`


feedbackvis

**Comment by Mynk96** [comment]
@chowyb Ready For Review!

**Comment by Mynk96** [comment]
@chowyb Please review

**Comment by Mynk96** [comment]
@chowyb Please Review!

**Comment by chowyb** [comment]
Issue still seems to be present on my end. Confirmed that I'm using the latest commit due to inspecting the eye icon and seeing `margin-right: 5px`.

**Comment by chowyb** [comment]
I'm not sure if adding margin is the right solution though. Here's how it currently looks at 10px margin.

padding

**Comment by Mynk96** [comment]
@chowyb the problem seems to disappear after merging of #7654. Please confirm!

**Comment by chowyb** [comment]
@Mynk96 tried on the latest commit `f83c973` on this branch (including `npm run build` and force-refreshing), still the same problem.

**Comment by Mynk96** [comment]
@chowyb Can you please see if these changes work.I'll fix the build errors

**Comment by chowyb** [comment]
@Mynk96 still not working for me. Also, by any chance, is it possible for you to replicate this behaviour (eg. by editing a comment, then editing the length of the instructor name longer or shorter as necessary until the bug occurs)?

**Comment by Mynk96** [comment]
@chowyb I tried all the possible combination to create the bug on various screen I could get, but everywhere it seems fine.

**Comment by chowyb** [comment]
@Mynk96 just tested on the latest commit of master ([`31edd44`](https://github.com/TEAMMATES/teammates/commit/31edd441ce2f678f155376fc37942f379a75d6c0)) and it appears that this is a problem even without the visibility icon. As such, it would be best to put this in another issue instead. I'll proceed with the rest of the review, ignoring this issue.

push

**Comment by Mynk96** [comment]
@chowyb let me revert the addition of `display:inline`. Please review after that.

File: src/main/webapp/WEB-INF/tags/shared/feedbackResponseCommentRow.tag [hunk 1]
```diff
@@ -27,7 +27,7 @@
 </c:choose>
 
 <li class="list-group-item list-group-item-warning" id="responseCommentRow-${divId}">
-    <div id="commentBar-${divId}">
+    <div id="commentBar-${divId}" style="display:inline;">
```
Comment (chowyb) [inline src/main/webapp/WEB-INF/tags/shared/feedbackResponseCommentRow.tag]: If we're putting the issue of misaligned buttons to another issue, this change can probably be removed from this PR.

File: src/main/java/teammates/ui/template/FeedbackResponseRow.java [hunk 2]
```diff
@@ -82,4 +93,52 @@ public String getResponseText() {
...
+                break;
+            default:
+                break;
+            }
+        }
+        String peopleCanViewString = peopleCanView.toString();
+        return removeEndComma(peopleCanViewString);
+    }
+
+    public String removeEndComma(String str) {
```
Comment (chowyb) [inline src/main/java/teammates/ui/template/FeedbackResponseRow.java]: There should not be a need to set this to `public`. Perhaps `private` would be better.

**Comment by Mynk96** [comment]
@chowyb Please Review!

**Comment by chowyb** [comment]
Also, as an additional comment, I realise that the changes to test files are quite minimal, even though additional feedback response comments were added to the sample course upon adding a new instructor from the admin page. Upon further observation, it seems that while the new tests added in #7511 do test for the responses in the sample course, it does not test for the comments in the sample course (due to being in the wrong view (question view).

Therefore, I would like to check if you intend to make comments viewable from the question view. If not, it should be possible to open a separate issue (classified as a testing issue instead of a comments issue) to fix the tests for adding a new instructor.

**Comment by Mynk96** [comment]
@chowyb Yes, That's also part of my proposal. Issue #2324. I've planned to work on this issue for last 4 weeks.It will automatically cover testing of comments in question view on completion.Currently, I am working on #7239 and will mostly submit a PR by today or tomorrow.

**Pull request merged**